src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -268,34 +268,45 @@ fn make_win_dist(
     let target_libs = find_files(&target_libs, &lib_path);
 
     // Copy runtime dlls next to rustc.exe
-    let dist_bin_dir = rust_root.join("bin/");
```

Comment:
I'm not sure the renamings are improvements (besides adding the missing `self_contained`s).

Reply:
I wanted to prefix paths derived from `rust_root` with `rust_`, and paths derived from `plat_root` with `plat_`.

I mixed up both on my first attempt and I wanted to make sure that this doesn't happen again, hence the renames.